### PR TITLE
make seq_assay_id check case insensitive

### LINKED ISSUE #435: Make validation of SEQ_ASSAY_IDs case-insensitive
The SEQ_ASSAY_IDs in the assay_information.yaml are being compared to the 'Sample Clinical Database' table (syn7517674), which converts the SEQ_ASSAY_IDs to all uppercase.  If the SEQ_ASSAY_IDs assay_information.yaml contain lowercase characters, they will be erroneously flagged.

### PR DESCRIPTION
**Pull request by hhunterzinck**
Fixes #435

### CODE REVIEW

File: genie_registry/assay.py [hunk 1]
```diff
@@ -141,7 +141,8 @@ def _validate(self, assay_info_df, project_id):
         warning = ""
 
         if process_functions.checkColExist(assay_info_df, "SEQ_ASSAY_ID"):
-            all_seq_assays = assay_info_df.SEQ_ASSAY_ID.unique()
+            all_seq_assays_raw = assay_info_df.SEQ_ASSAY_ID.unique()
```
Comment (thomasyu888) [inline genie_registry/assay.py]: Try .upper().unique()

**Comment by thomasyu888** [comment]
Need to add to tests

**Comment by thomasyu888** [comment]
In the future, please lint in a separate commit or PR, because it's hard to find the actual code changes.  Also I didn't run black on the tests due to the awful formatting of the context managers but it doesn't actually look too bad now.

That being said there are failing tests though

**Comment by hhunterzinck** [comment]
@thomasyu888 Tests were running fine before so linting did something.  Let me roll everything back and start from the beginning without linting

**Pull request closed without merging**